index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+
+<div class ="header">
```

Comment:
There's a `<header>` tag that can also be used inside the `<body>` tag.

But I think `<style>` tags are typically nested inside `<head>` tags like this:
```
<html>
  <head>
    <style></style>
  </head>
  <body></body>
</html>
```

Reply:
For the header tag, I don't think I know the good practice of what a header would be.  Should headings `<h1>`, `<h2>`, ... be used instead or is there a clear difference between header(s) and headings?

Reply:
What should be contained in the header?